crates/ruff_python_formatter/resources/test/fixtures/ruff/parentheses/call_chains.py

```diff
@@ -154,4 +154,6 @@
     five,
 )
 
-
+max_message_id = (
+    Message.objects.filter(recipient=recipient).order_by("id").reverse()[0].id
+)
```

Comment:
This works but i'd like to have it be part of the tests

```suggestion
)

max_message_id = (
    Message.objects.filter(recipient=recipient).order_by("id").reverse()[0].id()
)
```